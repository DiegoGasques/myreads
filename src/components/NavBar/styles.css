.navbar {
  padding: 10px 0;
  background: #4a4063;
}
