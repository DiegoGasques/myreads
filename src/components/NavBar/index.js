import React from "react";
import PropTypes from "prop-types";

import "./styles.css";

function NavBar({ children }) {
  return <nav className="navbar">{children}</nav>;
}

NavBar.propTypes = {
  children: PropTypes.node
};

export default NavBar;
