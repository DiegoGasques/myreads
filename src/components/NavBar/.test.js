import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import NavBar from "./index";

it("Should match NavBar component snapshot with children", () => {
  const wrapper = shallow(
    <NavBar>
      <h1>Navbar</h1>
    </NavBar>
  );
  expect(toJSON(wrapper)).toMatchSnapshot();
});
