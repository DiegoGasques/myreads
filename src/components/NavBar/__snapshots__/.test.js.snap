// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match NavBar component snapshot with children 1`] = `
<nav
  className="navbar"
>
  <h1>
    Navbar
  </h1>
</nav>
`;
